### PR TITLE
add training dropdown menu

### LINKED ISSUE #23569: Create "Training" drop down menu item on mooseframework.org
## Reason
A request has been made to create a new drop down menu labeled "Training". This will display links to training schedules, registration, etc

## Design
Create new markdown documents detailing the training schedule for the year(s). Create or move the registration instructions into this location.

Also: Clean up the Help page for contacting us: https://mooseframework.inl.gov/help/contact_us.html
There is information here that is no longer relvant (moose google group mailing list).

## Impact
A new landing page for folks interested in Training.

### PR DESCRIPTION
**Pull request by milljm**
The content which will reside in this menu has not yet been finalized.

Closes #23569

### CODE REVIEW

File: modules/doc/config.yml [hunk 1]
```diff
@@ -40,29 +40,29 @@ Extensions:
         repo: https://github.com/idaholab/moose
         google-cse: 000781241768298771085:pedtnlxdkx8
         menu:
+            Training:
+                Comming Soon: /empty.md
```
Comment (GiudGiud) [inline modules/doc/config.yml:44]:
```suggestion
                Coming Soon: /empty.md
```

File: modules/doc/config.yml [hunk 2]
```diff
@@ -40,29 +40,29 @@ Extensions:
...
+                Frequently Asked Questions: help/faq/index.md
+                Troubleshooting: help/troubleshooting.md
+                Help Forum: https://github.com/idaholab/moose/discussions
+                Contact Us: help/contact_us.md
             Documentation:
                 Application Usage: application_usage/index.md
                 Application Development: application_development/index.md
                 Framework Development: framework_development/index.md
                 Modules: modules/index.md
-                NCRC Applications: help/inl/applications.md
```
Comment (GiudGiud) [inline modules/doc/config.yml:55]: this should stay?

Comment (milljm) [reply modules/doc/config.yml:55]: I suppose we can leave it. Its duplicated under Getting Started, so I figure meh.

Comment (milljm) [reply modules/doc/config.yml:54]: added it back.

File: modules/doc/content/empty.md
Comment (GiudGiud) [inline modules/doc/content/empty.md:1]: what is this

Comment (milljm) [reply modules/doc/content/empty.md:1]: I need to remove... It was a placeholder. No longer needed.

**Comment by milljm** [comment]
Not quite ready yet...
Screenshot 2023-04-05 at 12 13 06 PM

**Review by GiudGiud — APPROVED** [review]

**Comment by milljm** [comment]
Documentation failed due to network related issues to civet... (pretty common). Please merge regardless.

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**